train_keras.py

```diff
@@ -470,6 +488,11 @@ def evaluate(model_root, model, images, image_size, num_batches, seed):
     model_name = FLAGS.output_model if FLAGS.output_model else build_model_name(
         FLAGS)
     logger.info('Model name {}'.format(model_name))
+    shared_data_path = os.path.join(os.environ['AZUREML_NATIVE_SHARE_DIRECTORY'])
```

Comment:
We should add another option for specifying an input zip file. Then, if that's specified, we will unzip that file into `--image_dir`